app/models/spree/order_decorator.rb

```diff
@@ -21,16 +21,25 @@
   # This removes "inverse_of: source" which breaks shipment adjustment calculations
   #   This change is done in Spree 2.1 (see https://github.com/spree/spree/commit/3fa44165c7825f79a2fa4eb79b99dc29944c5d55)
   #   When OFN gets to Spree 2.1, this can be removed
-  has_many :adjustments,
+  has_many :adjustments, -> { order "#{Spree::Adjustment.table_name}.created_at ASC" },
            as: :adjustable,
-           dependent: :destroy,
-           order: "#{Spree::Adjustment.table_name}.created_at ASC"
+           dependent: :destroy
 
   validates :customer, presence: true, if: :require_customer?
   validate :products_available_from_new_distribution, if: lambda { distributor_id_changed? || order_cycle_id_changed? }
   validate :disallow_guest_order
   attr_accessible :order_cycle_id, :distributor_id, :customer_id
 
+  # Removes Spree 2.1 additional email validation (currently failing every time)
+  # See: spree/core/validators/email.rb
+  _validate_callbacks.each do |callback|
+    if callback.raw_filter.respond_to? :attributes
+      callback.raw_filter.attributes.delete :email
+    end
+  end
```

Comment:
Is there an issue or a pending spec somewhere that helps makes sure the team don't forget to investigate this?